### PR TITLE
fixed tests that weren't running

### PR DESCRIPTION
**Pull request by AboudyKreidieh**
this was done in berkeleyflow but not merged. Moving it in here.

### CODE REVIEW

File: tests/fast_tests/test_vehicles.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def runTest(self):
 class TestObservedIDs(unittest.TestCase):
     """Tests the observed_ids methods, which are used for visualization."""
```
Comment (eugenevinitsky) [inline tests/fast_tests/test_vehicles.py]: Test not tests

File: tests/fast_tests/test_scenario_base_class.py [hunk 2]
```diff
@@ -535,7 +535,7 @@ def tearDown(self):
         # free data used by the class
         self.env = None
 
-    def runTest(self):
+    def test_even_start_pos_coverage(self):
         """
         Ensures that enough vehicles are placed in the network, and they cover
```
Comment (eugenevinitsky) [inline tests/fast_tests/test_scenario_base_class.py]: Ensure

File: tests/fast_tests/test_params.py [hunk 3]
```diff
@@ -71,7 +71,7 @@ def test_deprecated(self):
 class TestSumoLaneChangeParams(unittest.TestCase):
     """Tests flow.core.params.SumoLaneChangeParams"""
 
-    def runTest(self):
+    def test_lc_params(self):
         """Tests basic usage of the SumoLaneChangeParams object. Ensures that
```
Comment (eugenevinitsky) [inline tests/fast_tests/test_params.py]: Test

File: tests/fast_tests/test_params.py [hunk 4]
```diff
@@ -71,7 +71,7 @@ def test_deprecated(self):
 class TestSumoLaneChangeParams(unittest.TestCase):
     """Tests flow.core.params.SumoLaneChangeParams"""
```
Comment (eugenevinitsky) [inline tests/fast_tests/test_params.py]: Test

File: tests/fast_tests/test_params.py [hunk 5]
```diff
@@ -8,7 +8,7 @@
 class TestSumoCarFollowingParams(unittest.TestCase):
     """Tests flow.core.params.SumoCarFollowingParams"""
 
-    def runTest(self):
+    def test_params(self):
         """Tests that the various parameters lead to correct assignments in the
```
Comment (eugenevinitsky) [inline tests/fast_tests/test_params.py]: Test

**Comment by AboudyKreidieh** [comment]
fixed these, but there's a ton of other docstring mistakes in the tests folder. curious to find out why they aren't be caught by `pydocstyle`

**Pull request merged**